app/src/main/java/org/schabi/newpipe/util/TouchUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.schabi.newpipe.util;
```

Comment:
This, too, should be moved there

Reply:
I am not agree with your comment: `TouchUtils` may be used to provide in the future more methods which are not used by classes interacting with other apps.

Also, the current content of this class has nothing to do with external communication.

Reply:
Yeah ok, then `TouchUtils` should be moved to the spans package you mentioned below